src/main/java/co/rsk/federate/FederationProviderFromFederatorSupport.java

```diff
@@ -135,9 +143,58 @@ public Optional<Address> getRetiringFederationAddress() {
         return federatorSupport.getRetiringFederationAddress();
     }
 
+    @Override
+    public Optional<Federation> getProposedFederation() {
+        if (!federatorSupport.getConfigForBestBlock().isActive(RSKIP419)) {
+            return Optional.empty();
+        }
```

Comment:
this is not necessary. If RSKIP419 is not activated, the Bridge won't return anything

Reply:
If we include it we can avoid doing a callTx to the Bridge. Maybe is a better idea to leave it in place. wdyt?

Reply:
I think the less the powpeg knows about rskips, the better

Reply:
Interesting, maybe we can discuss this with the team